crutest/src/cli.rs

```diff
@@ -1071,9 +1100,12 @@ pub async fn start_cli_server(
      * If write_log len is zero, then the RegionInfo has
      * not been filled.
      */
+    let volume_info = VolumeInfo {
```

Comment:
Should we be pulling this info from the `volume` argument?

Reply:
It was that way because I could not figure out how to make in an `Option` and pass it back and forth between `process_cli_command` and `start_cli_server`.   Let me see if I can figure it out now

Reply:
I added an <Option> and I pass that around instead.